quizsoluation (1).ipynb

```diff
@@ -0,0 +1,181 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d8474a24-2bb4-468f-8bd1-280c0700b208",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "dec32fa4-24ab-4974-8c66-8ef3b07f7e9e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=open('contact.csv','w',newline='')"
```

Comment:
how to open this file to another way

Reply:
with open('contact.csv','w',newline='') as a: